app/controllers/api/v1/spaces_controller.rb

```diff
@@ -149,7 +147,19 @@ def destroy
   private
 
   def space_params
-    params.require(:space).permit(:phone, :name, :price_level, :provider_urn, :provider_url, hours_of_op: {}, address_attributes: [:id, :address_1, :address_2, :city, :postal_code, :country, :state], languages_attributes: [:id, :name], indicators_attributes: [:id, :name], photos_attributes: [:id, :url, :cover], reviews_attributes: [:id, :anonymous, :vibe_check, :rating, :content, :user_id])
+    params.require(:space).permit(
+      :phone, 
+      :name, 
+      :price_level, 
+      :provider_urn, 
+      :provider_url,
+      :latitude,
+      :longitude,
+      :category_aliases_attributes=> [:alias],
+      :address_attributes=> [:address_1, :address_2, :city, :postal_code, :country, :state], 
+      :indicators_attributes=> [:name], 
+      :reviews_attributes => [:anonymous, :rating, :content, :user_id]
```

Comment:
⭐

Reply:
Just fixed the failing space POST test (param formatting - facepalm)!